hazelcast-sql/src/test/java/com/hazelcast/jet/sql/SqlErrorTest.java

```diff
@@ -69,39 +64,28 @@ public void testMemberLeave() throws InterruptedException {
                 row(timestampTz(20), null, null)
         );
 
-        Semaphore semaphore = new Semaphore(0);
         Thread shutdownThread = new Thread(() -> {
-            try {
-                semaphore.acquire(); // wait until at least one row is received
-            } catch (InterruptedException e) {
-                e.printStackTrace();
-            }
+            // Waiting for a deployment of the job on both members.
+            assertTrueEventually(() -> {
+                assertEquals(1, instance1.getJet().getJobs().size());
```

Comment:
`JetService.getJobs` broadcasts to all members and adds the number of jobs on each. It's not sufficient.